docs/devices/cloud-protocol/pgps.schema.json

```
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/blob/saga/docs/devices/cloud-protocol/pgps.schema.json",
+    "title": "Asset Tracker v2 P-GPS request",
+    "description": "Describes the format which is used by the device to request P-GPS data.",
+    "type": "object",
+    "properties": {
+        "n": {
+            "type": "integer",
+            "description": "The number of predictions requested. Default is 42 (7 days, 6 predictions per day).",
+            "examples": [42]
+        },
+        "int": {
+            "type": "integer",
+            "description": "The time between predictions (in minutes). Defaults to 240 (4 hours).",
```

```suggestion
            "description": "The time between predictions, in minutes. Defaults to 240 (4 hours).",
```
It's easy to equate the two parentheticals, which isn't a correct reading. Taking out the first parenthesis fixes it.